Makefile

```diff
@@ -287,6 +287,17 @@ add-tags: | $(MULTIMOD)
 	@[ "${MODSET}" ] || ( echo ">> env var MODSET is not set"; exit 1 )
 	$(MULTIMOD) verify && $(MULTIMOD) tag -m ${MODSET} -c ${COMMIT}
 
+.PHONY: update-all-otel-deps
+update-all-otel-deps:
+	@[ "${GITSHA}" ] || ( echo ">> env var GITSHA is not set"; exit 1 )
+	@echo "Updating OpenTelemetry dependencies to ${GITSHA}"
+	@set -e; \
+		for dir in $(OTEL_GO_MOD_DIRS); do \
+			echo "Updating OpenTelemetry dependencies in $${dir}"; \
+			(cd $${dir} \
+			&& grep -v indirect go.mod | grep -o 'go.opentelemetry.io/otel\S*' | xargs -I {} -n1 $(GO) get {}@${GITSHA}); \
```

Comment:
Not updating the indirect dependencies may have transitive effects, right? I.e. https://github.com/google/go-licenses/issues/235.

Are there issues with updating these dependencies with this target?